db/migrate/20151125004633_create_pages.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePages < ActiveRecord::Migration
```

Comment:
Fixed the offending commit using a rebase interactive. The `db/schema.rb` should be up to date.